tests/feature_tests/features/metric_issue.feature

```diff
@@ -28,6 +28,10 @@ Feature: metric issues
     And the issue status connection_error is 'None'
     And the issue status parse_error is 'None'
 
+  Scenario: issue tracker options
+    When the client retrieves the issue tracker options
+    Then the issue tracker options are missing
```

Comment:
Huh, shouldn't it be there? Or is this the case because there is no jira linked/mocked ?

Reply:
Yes, we don't mock Jira during feature tests so the options will be missing. I'll add a comment.